src/_includes/certified/template-category.njk

```diff
@@ -0,0 +1,16 @@
+<div class="flex flex-col justify-between ff-blog container m-auto text-left max-w-4xl pt-8 pb-12 w-full ff-full-bg">
+    <div>
+    {% include "certified/common.njk" %}
+
+    <ul class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4 px-2 mt-6">
+        {%- asyncEach node in certifiedNodes -%}
+        {% if page.url.includes('/' + node.tags + '/')%}
+            {% include "certified/certified.njk" %}
+        {% endif %}
```

Comment:
This feels like a hacky solution to me, do you know how to clean it up? E.g. collections in the eleventy.js?

Reply:
I added the filter to eleventy.js and now a single template.njk works as a template for all, filtered and unfiltered.